docs/site/tutorials/authentication/Authentication-Tutorial.md

```diff
@@ -88,11 +88,11 @@ application, follow these steps:
 
    ```json
    {
-   "id": "1",
-   "email": "user1@example.com",
-   "password": "thel0ngp@55w0rd",
-   "firstName": "User",
-   "lastName": "One"
+     "id": "5dd6acee242760334f6aef65",
```

Comment:
@bajtos , let's avoid passing in an 'id' field with an ObjectID value to make it easy for the user. Mongo db will create the Object ID , and we will show later when `/users/me` is called.

Reply:
Just like the old example showed, it is also possible to pass in ids with values like "1" and "2", and this is easier for the user to do when creating new users.